specs/pvs/corla.pvs

```diff
@@ -132,14 +132,21 @@
 fafessl: THEORY
```

Comment:
Line 2036: 
```
  contest_cvr_table: TYPE =
    sequence[[contest, cvr]]

```
I don't see a purpose for organizing or storing things this way across counties.  All cross-county random selections (when we get to them) will be stratified by county, so that they don't end up having disjoint sets of random selections in a single county for multiple contests.